plutus-tx/test/Plugin/Spec.hs

```diff
@@ -392,10 +403,157 @@ evenMutual = plc @"evenMutual" (
         odd n = if n == 0 then False else even (n-1)
     in even)
 
+readCompiledCode :: PLC.KnownDynamicBuiltinType a => (a -> b) -> CompiledCode b -> b
```

Comment:
Can we stick the new stuff in another file? This one is already arguably too big.

Reply:
Sure, but I'm traveling today, so when I have time.